app/infrastructure/parsers/lms_parser.py

```
@@ -0,0 +1,301 @@
+import asyncio
+from typing import Optional, Callable, Awaitable
```

Ruff (0.12.2)

Import from `collections.abc` instead: `Awaitable`

Import from `collections.abc`

(UP035)